test/e2e_node/node_perf_test.go

```diff
@@ -127,11 +127,29 @@ var _ = SIGDescribe("Node Performance Testing [Serial] [Slow]", func() {
 		pod = f.PodClient().CreateSync(pod)
 		// Wait for pod success.
 		f.PodClient().WaitForSuccess(pod.Name, wl.Timeout())
```

Comment:
Do we want to remove the `WaitForSuccess` here?

Reply:
Thanks. patch incoming.